bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -172,6 +200,10 @@
 		<item-type>Switch</item-type>
 		<label>Super Cool</label>
 		<description>Start or stop Super Cooling</description>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Airconditioning</tag>
```

Comment:
This is a switch for a refrigerator or freezer to increase cooling. Airconditioning seems wrong in this context, or?

Reply:
Changing to `Switch.Enabled`